src/platform/Kconfig

```diff
@@ -353,4 +353,13 @@ config HAVE_AGENT
 	  with DMA based scheduling, where asynchronous interrupts
 	  can potentially starve the agent.
 
+config AGENT_TIME_PANIC
+	bool "Enable system agent time verification panic"
+	default n
```

Comment:
what about PANIC_ON_DELAY instead of AGENT_TIME_PANIC. This makes the config option pretty similar with the variable that triggers the panic on delay.

Reply:
AGENT_PANIC_ON_DELAY then? Because I'd still prefer to have a reference to the actual module responsible in the name

Reply:
@slawblauciak, yes looks better. Thanks!